roles/app/templates/nginx-vhost.conf.j2

```diff
@@ -53,7 +53,12 @@ server {
         try_files $uri /app_dev.php$is_args$args;
     }
 
-    location ~ ^/app_dev\.php(/|$) {
+    location ~ ^/app_(dev|test)\.php(/|$) {
```

Comment:
Nice solution!

The old test cookie solution was removed, but not completely reverted to its former state.

When I look at https://github.com/OpenConext/Stepup-Deploy/pull/72/files I see a modification to the pool config file. Shouldn't this change be reverted in this commit too?

Reply:
Removed the line from the pool config